recirq/algorithmic_benchmark_library.py

```diff
@@ -48,6 +48,7 @@ class AlgorithmicBenchmark:
     spec_class: Type['ExecutableSpec']
     executable_generator_func: Callable[[Any], 'QuantumExecutableGroup']
     configs: List['BenchmarkConfig']
+    description: Optional[str] = None
 
     def as_strings(self):
```

Comment:
Not related, but noticed it was missing return type.

Reply:
done